src/extension.ts

```diff
@@ -134,7 +134,7 @@ export function activate(context: vscode.ExtensionContext): void {
         }
 
         const createdApp = <IAzureNode<WebAppTreeItem>>await node.createChild();
-
+        await tree.refresh();
```

Comment:
It would be better to fix the UI package so that it inserts the node in the sorted position after creation finishes. Calling refresh like this will re-fetch _all_ resources from Azure, which is unnecessary

Plus fixing it directly in the UI package will fix it for everyone :)